src/components/parameter/FieldSelector.js

```diff
@@ -129,54 +178,26 @@ const FieldSelectorTree = ({ name, field, onChange, disabled }) => {
         )}
       </IconButton>
       {/* checkbox */}
-      <FormControlLabel
-        control={
-          // eslint-disable-next-line react/jsx-wrap-multilines
-          <Checkbox
-            id={buildFieldSelectorCheckbox(name)}
-            checked={selected}
-            onChange={handleCheckboxOnChange}
-            color="primary"
-            className={classes.checkbox}
-            indeterminate={
-              !selected &&
-              properties &&
-              anySelected({ type: 'object', properties })
-            }
-            disabled={disabled}
-          />
-        }
-        label={name}
-      />
-      {/* recursively render children */}
-      {properties && expanded && (
-        <div className={classes.shifted}>
-          {Object.entries(properties).map(([key, childField]) => {
-            return (
-              <FieldSelectorTree
-                key={key}
-                name={key}
-                field={childField}
-                onChange={(updatedField) => {
-                  // eslint-disable-next-line no-unused-expressions
-                  type.includes('object')
-                    ? onChange({
-                        ...field,
-                        properties: { ...properties, [key]: updatedField },
-                      })
-                    : onChange({
-                        ...field,
-                        items: {
-                          ...field.items,
-                          properties: { ...properties, [key]: updatedField },
-                        },
-                      });
-                }}
-                disabled={disabled}
-              />
-            );
-          })}
-        </div>
+      {
+        <FormControlLabel
+          control={
+            // eslint-disable-next-line react/jsx-wrap-multilines
```

Comment:
You just need to add `()` around the jsx and remove this eslint comment.

Reply:
I have an issue where `yarn prettier:write` removes those `()`... I didn't find a workaround...

Reply:
I guess you can just factor out the component then?